common/src/main/java/io/netty/util/concurrent/FastThreadLocal.java

```diff
@@ -134,13 +134,18 @@ public FastThreadLocal() {
      */
     @SuppressWarnings("unchecked")
     public final V get() {
-        final InternalThreadLocalMap threadLocalMap = InternalThreadLocalMap.get();
```

Comment:
was removing `final` intentional and for stylistic reasons?

Reply:
yeah I added it before when I needed it as I used it in the `Runnable` but as I moved this to an extra method I not need it anymore